### ISSUE TITLE
Separate dependencies from JS/CSS code

### ISSUE THREAD

**Issue by alexcasalboni** [danilop/ServerlessByDesign#5]
I would make clear what is a library/dependency and what's part of the repo. Eventually, we could add a bit of dependency management and injection (npm, bower, browserify, gulp, webpack, etc.).

**Comment by danilop** [comment]
I agree, the reason I included the source JS/CSS files for the dependacies was that I still wanted people to be able to clone the repo and start using it *locally*, without having to use a too such as npm/bower to install the dependencies.

**Comment by rts-rob** [comment]
PR #2 adds in browserify which I suggest is a good middle ground. If we upload `bundle.js` (as the PR does) then it's ready to run locally without any additional commands as soon as the user clones it, but we still get the modularity of browserify (which I used to avoid name collisions between the engines).

**Comment by alexcasalboni** [comment]
@fblecha @rts-rob I'm not necessarily suggesting to use CDNs for external libraries. I am suggesting to remove them from the repo and use a proper dependency management tool.
This way, you'd only install dependencies once at clone-time. Also, we'll be able to update libs versions over time without manually copying huge files.
